### ISSUE TITLE
Purpose of the Complex Contextual Chaining lookup

### ISSUE THREAD

**Issue by kmansourMT** [OpenType/opentype-layout#1]
Regarding the Complex Contextual Chaining, I would like to pose some basic questions:

What problem does this proposed lookup intend to solve? What new capabilities will it add to what the current offering? A description of the concept with graphic examples would be most appreciated. 
@mhosken

**Comment by behdad** [comment]
This proposal is not ready for review yet.

**Comment by kmansourMT** [comment]
We were invited by SIL to comment.

**Comment by behdad** [comment]
Oh, I didn't know that.  I'll try to write the rationale section for it next week, unless Martin wants to beat me to that.

**Comment by mhosken** [comment]
@kmansourMT Yes, but please do understand that this proposal is in an early stage and the rationale hasn't yet been written. So for now, can I suggest we keep our discussion to the implementation/details of the proposal and give Behdad the time he needs to get to the rationale at his convenience. The invitation was part of a general invitation to comment on everything in this repo. Different proposals are at different stages.
In short, it's fine to ask the rationale question and we acknowledge that a rationale for this proposal is still outstanding. But please don't write it off, yet, for lack of one. If when it is written it still fails to convince, then that's something worth discussing.
